src/components/NurseryWithdrawals/WithdrawalLogRenderer.tsx

```diff
@@ -20,82 +20,93 @@ const useStyles = makeStyles((theme: Theme) => ({
   },
 }));
 
-export default function WithdrawalLogRenderer(props: RendererProps<TableRowType>): JSX.Element {
+const WithdrawalLogRenderer = (formatter: { format: (num: number) => string }) => {
```

Comment:
made this a higher order function, causing most of the code below to be indented by 2 space.. showing up as modified..